portfolio/src/main/webapp/index.html

```diff
@@ -5,18 +5,56 @@
     <title>Lorem</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <link href="https://fonts.googleapis.com/css?family=Poiret+One&display=swap" rel="stylesheet">
   </head>
   <body>
+    <br/>
```

Comment:
These changes are duplicated from your other branch.  Can you please split out the new changes?  Otherwise it's pretty much impossible to review this

Reply:
I'll try comparing to the other branch instead of master